tasks/vm.py

```diff
@@ -44,6 +45,20 @@ def create_vm(
         agent_version=agent_version,
         extra_flags=extra_flags,
     )
+    _show_connection_message(full_stack_name)
+
+
+def _show_connection_message(full_stack_name: str):
+    outputs = tool.get_stack_json_outputs(full_stack_name)
+    connection = tool.Connection(outputs)
+    host = connection.host
+    user = connection.user
+
+    command = f"ssh {user}@{host}"
+    pyperclip.copy(command)
+    print(
+        f"\nYou can run the following command to connect to the host `{command}`. This command was copied to the clipboard\n"
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Yessss